facebookads/__init__.py

```diff
@@ -21,7 +21,10 @@
 from facebookads.session import FacebookSession
 from facebookads.api import FacebookAdsApi
 
+
+__version__ = FacebookAdsApi.SDK_VERSION
```

Comment:
I decided to keep it on the class, otherwise I got a cyclic import... (because the class needs it when building its headers)